src/en/developer-debug-dhx.md

```diff
@@ -1,13 +1,11 @@
 # DHX: A Customized Hook Debugging Environment Plugin
 
-[DHX](https://github.com/juju/plugins/blob/master/juju-dhx) is a Juju plugin
-that allows you to fully and automatically customise the machines created by
-Juju, making developing and debugging hooks as painless as possible.
-
-It is a drop in replacement for
-[`juju debug-hooks`](./developer-debugging.html#the-'debug-hooks'-command).
-It is still a plugin and not yet integrated into Juju itself, so currently
-considered beta quality. DHX is recommended for more advanced users who need to
+[DHX](https://github.com/juju/plugins/blob/master/juju-dhx) is a plugin --
+scripts that extend the functionality of Juju -- that allows you to fully and
```

Comment:
`that allows you` seems odd to me, `which allows you` sounds more natural*

*IANAEM